tools/metadata_loader/main.py

```diff
@@ -0,0 +1,185 @@
+import argparse
+import json
+import os
+from urllib.parse import quote
+
+import requests
+from azure.storage.blob import (  # pyright: ignore[reportMissingImports]
+    BlobServiceClient,
+)
+from dotenv import load_dotenv
+
+
+def get_config():
+    load_dotenv()
+    return {
+        "API_URL_BASE": os.environ.get("API_URL_BASE"),
+        "BLOB_STORAGE_ACCOUNT_URL": os.environ.get("BLOB_STORAGE_ACCOUNT_URL"),
+        "BLOB_STORAGE_SAS_KEY": os.environ.get("BLOB_STORAGE_SAS_KEY"),
+    }
+
+
+def call_get_datasources_api(url):
```

Comment:
I dont get the why of these call methods that are jsut one line why not call requests directly in the function?

Reply:
makes them easier to mock. If you have a better way, I'm all ears.